### DISCUSSION TITLE
Add support for feed handles

### DISCUSSION THREAD

**Discussion by kristiansp** [engram-design/FeedMe#927] [Ideas]
### Description

Especially when dealing with multiple feeds and utilizing events, it would be useful to give feeds (possibly optional) handles to reference them by, not only IDs. This would make it more readable when using events to tweak certain feeds and not others.

### Steps to reproduce

1. Create multiple feeds in Feed Me
2. Try accessing one specific one through use of events, and realize you have to reference the feed ID, where a named handle would make the code more easy to read and come back to.

### Additional info

- Plugin version: Any
- Craft version: Any

**Comment by FrancisMawn** [comment]
I came here to suggest the same thing. 

It'd be nice if the handle could also be used to add feed specific configurations instead of using the feed ID as key...
